includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,21 +1,271 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	private $data = [
+		"logo" => [
+			"header" => [
+				"type" => "link-image",
+				"href" => "http://fandom.wikia.com",
+				"image" => "wds/full_fandom_logo",
+				"title" => [
+					"type" => "text",
+					"value" => "Fandom",
+				]
+			]
+		],
+		"verticals" => [
+			"links" => [
+				[
+					"type" => "link-branded",
+					"brand" => "tv",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-tv",
+					],
+					"href" => "http://tv.wikia.com"
+				],
+				[
+					"type" => "link-branded",
+					"brand" => "games",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-games",
+					],
+					"href" => "http://games.wikia.com",
+				],
+				[
+					"type" => "link-branded",
+					"brand" => "movies",
+					"title" => [
+						"type" => "translatable-text",
+						"key" => "global-navigation-brandlink-vertical-movies",
+					],
+					"href" => "http://movies.wikia.com",
+				],
+			],
+		],
+		"wikis" => [
+			"header" => [
+				"type" => "link-branded",
```

Comment:
What do you think about changing this type to something else. In fact semantically and in the code it isn't a link - it's dropdown entry point.